### DISCUSSION TITLE
Eager to contribute

### DISCUSSION THREAD

**Discussion by MukeshKaswan1** [wagtail/gsoc#3] [General]
Hello everybody
 My name is Mukesh Kaswan , and I'm a computer science and engineering undergraduate student at the Punjab Technical University in India. I'm thrilled to participate in our organization’s GSoC initiative this year. When I came across the organization's project ideas, I realized I had to get involved.

I'm passionate about web development, have expertise with React, Material, UX , UI, JavaScript and NodeJS, and would also like to learn anything faster. I'm especially interested in working on the Wagtail  for "Dark theme for Wagtail admin", and after reading through the project concepts. I'm eager to learn and add to the open-source initiatives of the company. I'm thrilled to be a part of this team and can't wait to collaborate on the project and hone my skills.

If you have any tips or recommendations for someone who is new to the community, or if there are any particular steps I need to take to get started, please let me know. I am willing to learn about the process.
Thanks.

**Comment by thibaudcolas** [comment]
Welcome @MukeshKaswan1! We were running a bit behind and only shared this after your post – please make sure to take a look at  our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md). Unfortunately we’ve had to pull the "dark theme" project idea as it didn’t feel substantial enough for a GSoC project.

**Reply by MJCoder15** [reply]
Sir I am not able to find the page of ideas list which contains other projects details also. I went to GSoC's  website and clicked on "View IdeaList" but I am unable to find it now

**Reply by laymonage** [reply]
It has been moved to: https://github.com/wagtail/gsoc/blob/main/project-ideas.md

**Reply by MJCoder15** [reply]
Thanks for the Link👍🏽

**Reply by thibaudcolas** [reply]
Sorry about that @MJCoder15! I hadn’t realised we broke the link, have updated it now.

**Reply by MJCoder15** [reply]
No Worries 😅
